### ISSUE TITLE
An exception occurs when comparing CronTab object with None

### ISSUE THREAD

**Issue by sidahmed-malaoui** [josiahcarlson/parse-crontab#35]
If an we compare a `CronTab` object with `None`, an exception occurs:
```python
cron = crontab.CronTab("* * * * *")
if cron == None:
    pass
```
>AttributeError: 'NoneType' object has no attribute 'matchers'

The problem is in the definition of `CronTab.__eq__()`.
https://github.com/josiahcarlson/parse-crontab/blob/ccf931e1dce0178892a85843c319cf242095f425/crontab/_crontab.py#L388-L389

If `other` is not an instance of `CronTab`, we should directly return `False` without going further.

I can do a pull request for this.

**Comment by josiahcarlson** [comment]
That was not intended. Let me fix.

**Comment by josiahcarlson** [comment]
Just released version 0.22.9 with the fix. Thank you for the bug report.

**Comment by sidahmed-malaoui** [comment]
You're welcome dude. Thank you for your fast response.
